onnx/reference/ops/op_split_to_sequence.py

```diff
@@ -1,19 +1,41 @@
 # SPDX-License-Identifier: Apache-2.0
 # pylint: disable=C0200,W0221
 
-from .op_split import CommonSplit
+from onnx.reference.op_run import OpRun
 
 
-class SplitToSequence(CommonSplit):
+class SplitToSequence(OpRun):
+    def common_run(self, mat, split, axis, length):  # type: ignore
```

Comment:
IIUC, the input parameter `length` is not used here and can be omitted.

Reply:
It is used in line 15.

Reply:
But that value is defined in line 14, it's not the passed in parameter value

Reply:
Sorry, i did not see I was computing length in both methods. I removed the unnecessary code.